substrate/client/transaction-pool/src/fork_aware_txpool/fork_aware_txpool.rs

```diff
@@ -1017,12 +1097,13 @@ where
 				});
 		}
 
-		log::debug!(target: LOG_TARGET,
-			"fatp::extrinsics_included_since_finalized {} from {} count: {} took:{:?}",
-			at,
-			recent_finalized_block,
-			all_extrinsics.len(),
-			start.elapsed()
+		tracing::debug!(
+			target: LOG_TARGET,
+			at = ?at,
+			recent_finalized_block = ?recent_finalized_block,
+			extrinsics_count = all_extrinsics.len(),
+			took = ?start.elapsed(),
```

Comment:
we should be consistent and use `took` or `duration`.
You used `duration` previously and it seems better choice then `took`.

```suggestion
			target: LOG_TARGET,
			?at,
			?recent_finalized_block,
			extrinsics_count = all_extrinsics.len(),
			duration = ?start.elapsed(),
```